app/views/orders/show.html.erb

```diff
@@ -1,3 +1,5 @@
 <% @order.line_items.each do |line_item| %>
   <%= render line_item %>
 <% end %>
+
+<%= button_to 'Cancel Order', cancel_order_path, data: { turbo_confirm: 'Are you sure you want to cancel this order?' }, class: 'btn btn-danger' if  @order.Placed? %>
```

Comment:
let's make sure to never have methods with capital first letters

Reply:
ok